account_customer_wallet/models/res_partner.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 Coop IT Easy SCRL fs
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class Partner(models.Model):
+    _inherit = "res.partner"
+
+    customer_wallet_account_id = fields.Many2one(
+        comodel_name="account.account",
+        compute="_compute_customer_wallet_account_id",
+    )
+    # TODO: Is Monetary the correct field? What is behaviour with multiple
+    # currencies?
+    customer_wallet_balance = fields.Monetary(
+        string="Customer Wallet Balance",
+        currency_field="company_currency_id",
```

Comment:
no need to set this field. and to defined ``company_currency_id``.

we can define this field the same way odoo does : 
https://github.com/odoo/odoo/blob/12.0/addons/account/models/partner.py#L389